test_scripts/RC/OnRCStatus/commonOnRCStatus.lua

```diff
@@ -97,17 +97,27 @@ function m.getHMIAppIdsRC()
   return out
 end
 
-function m.registerRCApplication(pAppId)
+function m.registerRCApplication(pAppId, pAllowed)
   if not pAppId then pAppId = 1 end
-  local pModuleStatus = {
-    freeModules = m.getModulesArray(m.getAllModules()),
+  if not pAllowed then pAllowed = true end
```

Comment:
@GetmanetsIrina pAllowed - boolean parameter, and when you call this function like
 _common.registerRCApplication, { 1 , false }_
then this value will be changed to true